src/pre_defined_classes/simple_programs_taking_string_user_input/hard/q9/Origin.java

```diff
@@ -0,0 +1,43 @@
+package pre_defined_classes.simple_programs_taking_string_user_input.hard.q9;
+
+/*
+People all around the world use last names to identify their origins. Common last names include:"Silva" from Portugal
```

Comment:
This is just a very strange question, I am going to ask Bowen if we can just get rid of this one.